types/react-native/test/ui-manager.tsx

```diff
@@ -0,0 +1,4 @@
+import { UIManager } from 'react-native';
+
+UIManager.dispatchViewManagerCommand(1, 1, undefined);
```

Comment:
Your previous example in webview used null, not undefined. Are you sure undefined is supported?

Reply:
For those that look later, this is the type in core: https://github.com/facebook/react-native/blob/91f139b94118fe8db29728ea8ad855fc4a13f743/Libraries/ReactNative/NativeUIManager.js#L45

Reply:
Yea, in fact the hot fix for the issue that was caused by accidentally following these types switched to using undefined. https://github.com/react-native-community/react-native-webview/pull/886

In practice either works but this is a smaller delta in terms of changing the types and also feels more accurate for how that argument is meant to be used.

Reply:
`?foo` in Flow can be `null` or `undefined`, so really it should be both, but we haven’t done that consistently, if at all, it looks to be mostly `null` throughout the file. However, in this case `reactTag`, which is part of the same signature, is typed as `number | null`, so I think that in this case it should follow that to at least be consistent with other parameters within the same signature.